favorite-sites.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>My favorite websites</title>
+	<meta charset="utf-8">
+	<meta name="Description" content=This page show my favorite websites">

```

Comment:
you missed an opening quote. This should read
` 	<meta name="Description" content="This page show my favorite websites"> `